src/ucs/sys/sock.h

```diff
@@ -226,8 +225,7 @@ ucs_status_t ucs_socket_recv_nb(int fd, void *data, size_t *length_p,
  * @param [in]      err_cb          Error callback.
  * @param [in]      err_cb_arg      User's argument for the error callback.
  *
- * @return UCS_OK on success, UCS_ERR_CANCELED if connection closed,
- *         UCS_ERR_IO_ERROR on failure.
+ * @return UCS_OK on success, UCS_ERR_IO_ERROR on failure.
```

Comment:
Does `UCS_ERR_IO_ERROR` always mean that connection is closed? Can we have recoverable symptoms?